### PR TITLE
:fire: First draft of new documentation

### PR DESCRIPTION
**Pull request by pl12133**
TODO:

 - [x] Change `href` to point to upstream repo instead of forked repo.

### CODE REVIEW

File: docs/zimlet-templates.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+## Zimlet Templates
+
+Using Zimlet templates you can take full control of scaffolding your next Zimlet.
+
+```sh
+$: zimlet create default my-project
+```
+
+The above command pulls the template from [billneff79/zimlet-default-template](https://github.com/billneff79/zimlet-default-template) - the default template.
```
Comment (silentsakky) [inline docs/zimlet-templates.md]: https://github.com/Zimbra/zm-x-zimlet-template-default

**Comment by pl12133** [comment]
Most pages have moved into the Wiki, so I have pruned down the `README` to reflect that.

**Comment by pl12133** [comment]
Poking @billneff79 @silentsakky, think this is good to merge for the README changes?

File: docs/zimlet-templates.md (hunk 1 above)
Comment (silentsakky) [reply docs/zimlet-templates.md]: is there a difference between https://github.com/billneff79/zimlet-default-template and https://github.com/Zimbra/zm-x-zimlet-template-default?

**Pull request merged**